core/controlplane/config/templates/stack-template.json

```diff
@@ -203,12 +203,13 @@
                   "Resource": [ "*" ]
                 },
                 {{end}}
-                {{if .Experimental.ClusterAutoscalerSupport.Enabled}}
+                {{if .Addons.ClusterAutoscaler.Enabled}}
```

Comment:
Done for controller nodes.
I'm still unsure if we'd want to support the customization to schedule CA to worker nodes https://github.com/kubernetes-incubator/kube-aws/pull/629#discussion_r117418053